homeassistant/components/bsblan/climate.py

```diff
@@ -60,8 +60,7 @@ async def async_setup_entry(
                 data.static,
                 entry,
             )
-        ],
-        True,
+        ]
```

Comment:
If you're feeling really fancy, you can now remove the square brackets to avoid creating a separate list

Reply:
This will not work, it must be an `Iterable`

Reply:
oh good morning, I thought it was a for loop but didn't check